satpy/readers/abi_l1b.py

```diff
@@ -38,21 +38,27 @@ class NC_ABI_L1B(NC_ABI_BASE):
     def get_dataset(self, key, info):
         """Load a dataset."""
         logger.debug('Reading in get_dataset %s.', key['name'])
-        radiances = self['Rad']
-
-        if key['calibration'] == 'reflectance':
-            logger.debug("Calibrating to reflectances")
-            res = self._vis_calibrate(radiances)
-        elif key['calibration'] == 'brightness_temperature':
-            logger.debug("Calibrating to brightness temperatures")
-            res = self._ir_calibrate(radiances)
-        elif key['calibration'] != 'radiance':
-            raise ValueError("Unknown calibration '{}'".format(key['calibration']))
+        # For raw cal, don't apply scale and offset, use true file counts
+        if (key['calibration'] == 'counts'):
+            radiances = self.nc['Rad']
```

Comment:
I'm not a huge fan of something being called radiances and meaning two different things depending on the calibration. Others may disagree with me but what about a shortcut here like:

```
if key['calibration'] == 'counts':
    return self._raw_calibrate(self.nc['Rad'])
```

Then you can remove the `'counts': self._raw_calibrate` entry in the dict below.

Open for discussion, just a feeling.